src/Wallabag/CoreBundle/Repository/SiteCredentialRepository.php

```diff
@@ -2,17 +2,22 @@
 
 namespace Wallabag\CoreBundle\Repository;
 
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Wallabag\CoreBundle\Entity\SiteCredential;
 use Wallabag\CoreBundle\Helper\CryptoProxy;
 
 /**
  * SiteCredentialRepository.
  */
-class SiteCredentialRepository extends \Doctrine\ORM\EntityRepository
+class SiteCredentialRepository extends ServiceEntityRepository
 {
     private $cryptoProxy;
 
-    public function setCrypto(CryptoProxy $cryptoProxy)
```

Comment:
removing this is technically a BC break, but I think it was added just to inject the crypto proxy.
if you want I can keep it

Reply:
I think I used it because I wasn't able to inject it using the constructor of the repo, removing it is fine for me 👍🏼